docs/content/2.components/1.nuxt-img.md

```diff
@@ -44,6 +44,16 @@ Otherwise path that is expected by provider that starts with `/` or a URL.
 
 For image optimization when using external urls in `src`, we need to whitelist them using [`domains`](/api/options#domains) option.
 
+### `placeholder`
+
+Display a placeholder image before the actual image is fully loaded.
+
+You can either provide your own placeholder image or simply add the `placeholder` attribute and one will be automatically generated.
+
+```html
+<nuxt-img src="/nuxt.png" placeholder />
```

Comment:
Would you please also documents string, number and array use cases for placeholder and link to the roadmap issue?